tests/test_processor.py

```diff
@@ -167,16 +167,16 @@ def test_processor_samples_max_shots():
         samples = p.samples(max_samples, max_shots)
         assert len(samples['results']) == min(max_samples, max_shots)
 
-    p = Processor(Clifford2017Backend(), 4, Source(losses=.92))
+    p = Processor(Clifford2017Backend(), 4)
     p.add(0, catalog['postprocessed cnot'].build_processor())
     p.with_input(BasicState([0, 1, 0, 1]))
     max_samples = 100
     result_len = {}
-    for max_shots in [400, 2_000, 10_000]:
+    for max_shots in [100, 500, 2_000]:  # Success proba = 1/9  --> AVG 900 shots to get 100 samples
```

Comment:
is  `proba` a typing error or did you mean to abbreviate `probamplitude` ?

Reply:
It was for "probability" but I'll change it